cmd/dependabot/internal/cmd/test.go

```diff
@@ -49,6 +49,7 @@ var testCmd = &cobra.Command{
 			PullImages:    pullImages,
 			Timeout:       timeout,
 			UpdaterImage:  updaterImage,
+			UseStdout:     false,
```

Comment:
I'm not too familiar with the ergonomics of Golang, so please forgive my ignorance 😅 

Could we pass around the output file rather than a boolean?

Reply:
That's fair, I knew this code needed a refactor from this but the one I made got a little too large so I went with the bool. Let me try again with an `io.Writer` and see how that goes. 

I think the RunParams is starting to turn into a code smell, eventually I'll get around to fixing that up!